oss-ui/docker-image/Dockerfile.build

```diff
@@ -0,0 +1 @@
+FROM node:20.17
```

Comment:
Is this dockerfile needed if it's just an import of node?

Reply:
I don't actually know, I dumbly copied this stuff over from manager later on friday.

Reply:
Removed the container and am just using node.